code/modules/clothing/suits/bio.dm

```diff
@@ -21,7 +21,7 @@
 	clothing_flags = THICKMATERIAL | SHOWEROKAY
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	slowdown = 1
-	allowed = list(/obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/pen, /obj/item/flashlight/pen, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray)
+	allowed = list(/obj/item/tank/internals, /obj/item/tank/internals/plasmaman, /obj/item/pen, /obj/item/flashlight/pen, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray)
```

Comment:
This requires a //NSV13 tag

Reply:
Would I still need one if I do the same code change on Beestation-Hornet?

Reply:
Nope, but link the PR.

Reply:
Alright I've done it https://github.com/BeeStation/BeeStation-Hornet/pull/7281